### ISSUE TITLE
Bug on search options box

### ISSUE THREAD

**Issue by claudiobrandt** [bonny/WordPress-Simple-History#148]
There seems to be a bug on SH's search options' box where the placeholder string "All users" on the Users line remain visible after one opens a drop-down menu. When I click on the Log Levels drop-down, one of the menu items, "Notice", seems to superimpose over the string "All users". Also, that item cannot be clicked on properly at first, and when sliding down the box, the item can be clicked on, but the result is not the expected. Please see this screencast: 

[http://youtu.be/NfsowwgkehY?hd=1](http://youtu.be/NfsowwgkehY?hd=1)

**Comment by bonny** [comment]
Thanks for the bug report. I can however not reproduce this. Do you have any other plugins installed? Perhaps another plugin that also is using Select2 to create dropdowns?

**Comment by claudiobrandt** [comment]
You're welcome! Well, I tested again using the Troubleshoot plugin, which deactivates all plugins and switches the theme to the default theme, and the problem is gone. I tried to re-enable several plugins to find which one in causing this issue, but I couldn't find it. I have no idea which plugins might use Select2, but I tested activating my theme, Kirki Toolkit, Visual Composer (WP Bakery Page Builder), ACF, and a few others, but the issue did not come up. 

Well, since this only affects my WP backend, and not my frontend (or that of other users of Simple History), I've given up finding the plugin that might be causing it. Thanks.
